source/whats-new.txt

```diff
@@ -37,6 +38,20 @@ Learn what's new by version:
 * :ref:`Version 1.1 <kafka-connector-whats-new-1.1>`
 * :ref:`Version 1.0 <kafka-connector-whats-new-1.0>`
 
+.. _kafka-connector-whats-new-1.11.2:
+
+What's New in 1.11.2
+--------------------
+
+- Fixed a bug in which unsuccessful attempts to retrieve items from a change stream were logged as
+  INFO-level instead of ERROR-level events. To learn more about this fix, see the
+  `KAFKA-396 <https://jira.mongodb.org/browse/KAFKA-396>`__ JIRA issue.
```

Comment:
```suggestion
- Fixed a bug in which unsuccessful attempts to retrieve items from a change stream were logged at the
  ``INFO`` level instead of at ``ERROR`` level. To learn more about this fix, see the
  `KAFKA-396 <https://jira.mongodb.org/browse/KAFKA-396>`__ JIRA issue.
```